ShovIP/app/src/main/java/com/example/shovip/DialPadFragment.kt

```diff
@@ -0,0 +1,76 @@
+package com.example.shovip
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import com.example.shovip.databinding.FragmentDialPadBinding
+
+/**
+ * A simple [Fragment] subclass as the default destination in the navigation.
+ */
+class DialPadFragment : Fragment() {
+    private var binding by autoCleared<FragmentDialPadBinding>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentDialPadBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    @SuppressLint("SetTextI18n")
+    private fun loadData(){
+        val act = this.activity
+        val sharedPreferences = act!!.getSharedPreferences("sharedPrefs", Context.MODE_PRIVATE)
+        with(binding){
+            account.text = sharedPreferences.getString("uUSERNAME",null) + "@" + sharedPreferences.getString("DOMAIN",null)
```

Comment:
@shovian 

If there are no settings (yet), this will display `null@null`

![Screenshot_20210805-112415](https://user-images.githubusercontent.com/3959480/128281254-5ae4b4c8-5bd3-473d-b44f-d867b85cd75d.png)

Please fix.

Reply:
(it would be best if it displayed something like "No account" if either username or domain are missing)